### ISSUE TITLE
Metadtaloader from Metadata content (more a request than an issue)

### ISSUE THREAD

**Issue by robertgins** [Sustainsys/Saml2#988]
I need the ability to load metadata directly to an identity provider. the identity provider has a constructor that works fine , however the metadata loader load from an xml dictionary is not a publicly accessibly method. It would be very nice to have this method in the metadataloader class marked as public 
 **static MetadataBase Load(XmlDictionaryReader reader)**

thanks

**Comment by explunit** [comment]
Would #736 meet your needs once it is merged?

**Comment by robertgins** [comment]
Yes that is essentially what I did, After I made the Load function public, I converted the stream to a XmlDictionaryReader with CreateTextReader, and passed it to the Load function. Works great, so if a similar stream oriented overload was available, it would work fine.

**Comment by explunit** [comment]
ok thanks, will close this issue then in favor of 736
